app/src/main/java/com/zeapo/pwdstore/PasswordStore.kt

```diff
@@ -423,19 +418,18 @@ class PasswordStore : AppCompatActivity(R.layout.activity_pwdstore) {
      * is true if the permission has been granted.
      */
     private fun hasRequiredStoragePermissions(checkLocalRepo: Boolean = false): Boolean {
+        val cloning = supportFragmentManager.findFragmentByTag("ToCloneOrNot") != null
```

Comment:
The control flow around this is excruciatingly painful so I have resorted to worsening the mess.